core/modules/filters/debug-log.js

```diff
@@ -0,0 +1,36 @@
+/*\
+title: $:/core/modules/filters/debug-log.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator for logging the current list and an optional variable value to the console
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+/*
+Export our filter function
+*/
+exports["debug-log"] = function(source,operator,options) {
+	var results = [], data = {};
+
+	if (operator.operand)
+		data[operator.operand] = options.widget.getVariable(operator.operand,{defaultValue:""});
+
+	source(function(tiddler,title) {
+		results.push(title);
+	});
+
+	console.group("filter-debug");
```

Comment:
may be the group should also be named `debug-log` so we can see a connection

Reply:
That sounds sensible.